parts/k8s/manifests/kubernetesmaster-kube-controller-manager-custom.yaml

```diff
@@ -14,6 +14,9 @@ spec:
       imagePullPolicy: IfNotPresent
       command: ["/hyperkube", "controller-manager"]
       args: [<args>]
+      env:
+      - name: AZURE_ENVIRONMENT_FILEPATH
+        value: "/etc/kubernetes/azurestackcloud.json"
```

Comment:
Variable `AZURE_ENVIRONMENT_FILEPATH` is set in the custom data script.

Is it possible to use the variable value (`$AZURE_ENVIRONMENT_FILEPATH`) instead typing the same string twice?

I fear we can introduce regressions in the future if someone changes one string and not the other.

Reply:
Never mind, I looks like this is not possible

Reply:
It seems like we should consider an Azure Stack-specific controller manager spec, or at least only expose this environment variable if we're in an Azure Stack context.

(And we can figure out a way to source the filename from a common place as well, but let's figure out how to expose Azure Stack-specific stuff in an Azure Stack-exclusive context first.)

Reply:
@jackfrancis, if I understand correctly you comment, I think Hongbin has already addressed the concern. He has already introduced an AzS specific controller manager:

`kubernetesmaster-kube-controller-manager- **custom** .yaml`